octo-app/src/app/assign-members-menu/assign-members-menu.component.ts

```diff
@@ -9,14 +12,52 @@ import { ActivatedRoute, Params } from '@angular/router';
 export class AssignMembersMenuComponent implements OnInit {
   boardID: number;
 
-  constructor(private route: ActivatedRoute) { }
-  
+  constructor(private cookieService: CookieService, private assignMembersService: AssignMembersService, private route: ActivatedRoute) { }
+
+  dropdownList = [];
+  selectedItems = [];
+  dropdownSettings = {};
   ngOnInit() {
     this.route.params.subscribe(
-       (params: Params) => {
+      (params: Params) => {
         this.boardID = params["id"]; //grab the board ID
       }
     )
+    var cookie = this.cookieService;
```

Comment:
I am still flabbergasted why var cookie here works but we can't reach the component scope in a similar fashion.